specs/order_spec.rb

```diff
@@ -78,19 +78,28 @@
   end
 end
 
-# TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+describe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
+      Grocery::Order.all.must_be_kind_of Array
     end
 
     it "Returns accurate information about the first order" do
-      # TODO: Your test code here!
-    end
 
+      Grocery::Order.all[0].id.must_equal 1
+
+      products = {"Slivered Almonds" => 22.88, "Wholewheat flour" => 1.93, "Grape Seed Oil" => 74.9}
+      Grocery::Order.all[0].products.must_equal products
```

Comment:
This test looks good. One change you might make is to only call `Grocery::Order.all` once, and save the result in a local variable. Something like:

```ruby
# Arrange
id = 1
products = {"Slivered Almonds" => 22.88, "Wholewheat flour" => 1.93, "Grape Seed Oil" => 74.9}

# Act
first_order = Grocery::Order.all[0]

# Assert
first_order.id.must_equal id
first_order.products.must_equal products
```

This has the added benefit of making the _Arrange_, _Act_ and _Assert_ steps very distinct.